.github/workflows/pospell-on-demand.yml

```
@@ -0,0 +1,43 @@
+name: pospell
+on:
+  issue_comment:
+    types:
+      - edited
+      - created
+
+jobs:
+  fix-spell:
+    name: Fix spelling
+    runs-on: ubuntu-latest
+    if: ${{ github.event.issue.pull_request && github.event.comment.body == 'pospell-fix' }}
+    steps:
+      - name: Confirmar ejecucion
+        continue-on-error: true
+        run: |
+          curl ${{github.event.comment.url}}/reactions \
+            -X POST \
+            -d '{"content":"+1"}' \
+            -H "Accept: application/vnd.github.squirrel-girl-preview+json" \
+            -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}"
+      - uses: actions/checkout@v3
+      - uses: awalsh128/cache-apt-pkgs-action@latest
+        with:
+          packages: hunspell hunspell-es gettext
+          version: 1.0
+      - name: Preparar Python v3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+          cache: "pip"
+      - name: Instalar dependencias
+        run: python -m pip install -r requirements.txt
+      - name: Ejecutar Powrap
+        run: powrap --quiet **/*.po
+      - name: Revisar con Pospell
+        run: python scripts/check_spell.py
+        continue-on-error: true
+      - name: Commit & Push changes
```

Este workflow es bueno si `powrap`/`pospell` son confiable, y siempre quieres commitar directamente.
Pero estaba pensando que tambien podrias haber un comando `pospell-check` y usar:
```suggestion
      - name: Commit & Push changes
      - if: ${{ failure() && github.event.comment.body == 'pospell-fix' }}
```
para ejecutar este step solo si el comando es `pospell-fix`.

---

Por el comentario de @rtobar , creo que lo mejor sí será quitar pospell, para directamente dedicar la acción al `powrap`.